### ISSUE TITLE
Azure AI Immersive Reader Token Expired every time

### ISSUE THREAD

**Issue by SampathChintala** [microsoft/immersive-reader-sdk#526]
I setup an Azure Immersive Reader SDK for use in an ASP.Net MVC application using the Microsoft Learn tutorials. This all worked fine but now have an issue where the token is always shown as expired expired. I have given correct TenantId, ClientId, ClientSecret and Subdomain. Can some one help me on this.
<img width="485" alt="TokenExpire" src="https://github.com/user-attachments/assets/5ec0e7bb-4538-4a69-906b-64ab536eac26">

**Comment by cristobal-buenrostro** [comment]
Hi @SampathChintala 

please double check that you are getting the token in right way; [here is](https://learn.microsoft.com/en-us/azure/ai-services/immersive-reader/quickstarts/client-libraries?pivots=programming-language-csharp#update-the-controller-to-acquire-the-token) our nodeJS code sample where you can check the api call.
And you can also verify the token is passed correctly to the launchAsync function. See our sample [here](https://learn.microsoft.com/en-us/azure/ai-services/immersive-reader/quickstarts/client-libraries?pivots=programming-language-csharp#update-the-controller-to-acquire-the-token):

**Comment by cristobal-buenrostro** [comment]
Hi @SampathChintala , anything else we can help here?

**Comment by v-rmanda** [comment]
Closing the issue as no response is received from the customer.

**Comment by antoniu86** [comment]
hello @cristobal-buenrostro . I have the same issue. I get TokenExpired, but the timestamp seems to be ok. I have the integration with Immersive Reader for a long time and everything was fine when it was set using OAuth with client id/secret pair. now it seems that the Keys and Endpoints area of the Azure app, sets key1 and key2 and based on the docs I found out that one needs to be used and the code for the request for token needs to be changed. I managed to adapt the code and receive a JWT token and when decoded it seems to look ok. I also updated the CDN js sdk url for IReader from 1.0.0 to 1.4.0. but each time I start the request to load the IReader I get the Token Expired error.
